app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -399,7 +399,7 @@ def retro_fail_enrollment(enrollment, response)
     )
     analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_email_initiated(
       **email_analytics_attributes(enrollment),
-      email_type: 'Failed',
+      email_type: 'Failed unsupported secondary ID',
```

Comment:
my one thought with this is that there isn't really a separate version of the failed email for unsupported ids. We have a standard failure email (which is the one we're sending in this case) and a failure b/c of suspected fraud.

Reply:
yeah, I was hesitant at first, but noticed that we log `'Failed unsupported ID type'` in `handle_unsupported_id_type` above, and think this message would be useful for filtering when searching logs.